src/grid.tsx

```diff
@@ -1032,6 +1033,14 @@ export default class Grid {
   public removeRow(rowKey: RowKey, options: OptRemoveRow = {}) {
     const { treeColumnName } = this.store.column;
 
+    const editingAddr = this.store.focus.editingAddress;
+    if (editingAddr) {
+      const { rowKey, columnName } = editingAddr;
+      const cellVal = this.getValue(rowKey, columnName) || '';
+      this.finishEditing(rowKey, columnName, cellVal.toString());
+      rerender();
```

Comment:
Thank you for your PR. But it's need to be fixed. Check as below :)

1. Calling `rerender` function looks inefficient for removing the editing cell.
* preact render the DOM element async. So you can control the timing using setTimeout function (refer the `resetData` function)

2. move the source code to `removeRow` function in `dispatch/data.ts` for maintaining the source code.

3. Only remove the editing layer and focusing layer when removed rowKey is same as editing cell's rowKey

Reply:
oh I didn't know renderer running asynchronously. 
thanks for your information. i will push again.